ipapython/dogtag.py

```diff
@@ -228,8 +230,15 @@ def _httplib_request(
         logger.debug("httplib request failed:", exc_info=True)
         raise NetworkError(uri=uri, error=str(e))
 
+    encoding = res.getheader('Content-Encoding')
+    if encoding == 'gzip':
+        # note: gzip.decompress available in Python >= 3.2
+        http_body = gzip.decompress(http_body)  # pylint: disable=no-member
```

Comment:
This will fail with an attribute error on Python 2. Debian still uses Python 2.7. This should do the trick:

```
import gzip
import io
import six

if six.PY3:
    gzip_decompress = gzip.decompress
else:
    def gzip_decompress(data):
        with GzipFile(fileobj=io.BytesIO(data)) as f:
            return f.read()
```

Code based on https://github.com/python/cpython/blob/3.6/Lib/gzip.py#L527